salt/modules/openscap.py

```diff
@@ -55,6 +56,115 @@ def error(self, message, *args, **kwargs):
 }
 
 
+def xccdf_eval(xccdffile, ovalfiles=None, **kwargs):
```

Comment:
I would prefer to not allow `**kwargs` and be more explicit about the kwargs we want to allow.

Reply:
Done